### PR TITLE
Choose times for Automatic Dark Mode

### PR DESCRIPTION
**Pull request by nestordominguez**
## PR Checklist

Please check all that apply to this PR using "x":

- [x ] I have checked that this PR is not a duplicate of an existing PR (open, closed or merged)
- [ x] I have checked that this PR does not introduce a breaking change
- [ ] This PR introduces breaking changes and I have provided a detailed explanation below

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [x ] Feature
- [ ] Code style update (formatting)
- [ ] Refactoring (no functional changes)
- [ ] Documentation changes
- [ ] Other - Please describe:

## Fixes

Issue Number: https://github.com/lbryio/lbry-desktop/issues/1922

## What is the current behavior?
The settings dark mode enabled have a starter time and end time fixed(from 9 pm to 8 am)

## What is the new behavior?
With this new changes now you can select the start time and the end time (hours and minutes)

### CODE REVIEW

**Comment by nestordominguez** [comment]
this pr need update lbry-redux, I pushed other pr with the constant needed

**Comment by seanyesmunt** [comment]
Can you change this to add the constants in this repo?

File: src/ui/scss/component/_settings.scss [hunk 1]
```diff
@@ -0,0 +1,9 @@
+.select-hours {
```
Comment (seanyesmunt) [inline src/ui/scss/component/_settings.scss]: This shouldn't need new styles. Please try to look at existing form elements to see how we can do this without adding new css

**Comment by seanyesmunt** [comment]
Thanks for the PR! I'll test this out/look at it closer tomorrow.

File: src/ui/scss/component/_settings.scss (hunk 1 above)
Comment (nestordominguez) [reply src/ui/scss/component/_settings.scss]: I will look

**Comment by nestordominguez** [comment]
Ready, I pushed a new commit

**Pull request merged**